src/wasm/literal.cpp

```diff
@@ -72,20 +118,25 @@ double Literal::getFloat() const {
   }
 }
 
-int64_t Literal::getBits() const {
+void Literal::getBits(uint8_t (&buf)[16]) const {
   switch (type) {
```

Comment:
Not sure what you mean by raw pointer - of what type? (void*?)

Perhaps there could be `getBits(uint8_t& x)` which would write a 1-byte value, and assert the value fits into one byte, and likewise overrides with `uint16&` etc., but what would be the type for the simd variant, though?

Reply:
Yes, the current code now has `getBits` take a `void*`. You can see how this is used in the implementation of `less` at the bottom of literal.h. Overriding `getBits` for each size seems similar to removing the type asserts from `geti32()` and `geti64()`, which would also serve most of the same usecases. `getBits` would then become `getv128` since it wouldn't be useful otherwise.

Reply:
Oh, I'm not a fan of the `void*` approach... that seems too risky. Overriding for each size would ensure that we write to the proper type, which we'd lose with `void*`?

Reply:
This is fixed now.